tasks/main-blivet.yml

```diff
@@ -0,0 +1,130 @@
+---
+- name: get a list of rpm packages installed on host machine
+  package_facts:
+    manager: "auto"
+  when: false
+
+- name: make sure blivet is available
+  package:
+    name: "{{ blivet_package }}"
+    state: present
+
+##
+## This is all to fill in missing defaults in the list of pools (and their volumes).
+##
+- name: initialize internal facts
+  set_fact:
+    pools: []
+    volumes: []
+    all_vols: []
+    vols: []
+    vols_by_pool: []
+    vol_pools: []
```

Comment:
there is no such thing as internal facts. All facts are global, so please use a prefix to avoid namespace pollution. If they are meant to be internal, use a `_storage_` prefix.